addons/radar/XEH_postInit.sqf

```diff
@@ -55,7 +55,9 @@ if (isClass(configfile >> "CfgPatches" >> "ace_finger")) then {
     }]  call CBA_fnc_addEventHandler;
 };
 
-if (isClass (configFile >> "CfgPatches" >> "tfar_core")) then {
+private _tfar = isClass (configFile >> "CfgPatches" >> "tfar_core");
```

Comment:
Var declaration not really needed as they are not used afterwards. Combining both conditions should be still readable enough.

Suggestion:
```sqf
if (isClass (configFile >> "CfgPatches" >> "tfar_core") || {isClass (configFile >> "CfgPatches" >> "task_force_radio")}) then {
```